### PR TITLE
Fixes https://github.com/fastlane/examples/issues/5

### PR DESCRIPTION
**Pull request by fulldecent**

### CODE REVIEW

**Comment by KrauseFx** [comment]
Thanks for your contribution @fulldecent :+1:

File: docs/Advanced.md [hunk 1]
```diff
@@ -209,3 +209,21 @@ The documentation was moved to [Appfile.md](https://github.com/KrauseFx/fastlane
...
+    /fastlane/Error*.png
+    # Deliver temporary preview output
+    /fastlane/Preview.html
+    # Snapshot generated screenshots
+    /fastlane/screenshots/*/*-portrait.png
+    /fastlane/screenshots/*/*-landscape.png
+    /fastlane/screenshots/screenshots.html
+    # Frameit generated screenshots
+    /fastlane/screenshots/*/*-portrait_framed.png
+    /fastlane/screenshots/*/*-landscape_framed.png
```
Comment (0oneo) [inline docs/Advanced.md]: why are the lines are preceding blanks??

Comment (0oneo) [inline docs/Advanced.md]: not convinience for copy/paste

Comment (KrauseFx) [inline docs/Advanced.md]: Not sure what you mean, you can just copy them from here https://github.com/fastlane/fastlane/blob/bfc7a6e277bbede8b15c1d89a31ebdd7c5e90b63/docs/Advanced.md#gitignore

**Pull request merged**